eng/Versions.props

```diff
@@ -25,6 +25,8 @@
     <MicrosoftBclJsonSourcesPackageVersion>4.6.0-preview.19109.6</MicrosoftBclJsonSourcesPackageVersion>
     <MicrosoftCSharpPackageVersion>4.6.0-preview.19109.6</MicrosoftCSharpPackageVersion>
     <MicrosoftWin32RegistryPackageVersion>4.6.0-preview.19109.6</MicrosoftWin32RegistryPackageVersion>
+    <SystemComponentModelPackageVersion>4.3.0</SystemComponentModelPackageVersion>
```

Comment:
The corefx team is no longer producing nighties of this assembly. This should be moved down into the "manual" section (look for the comment that says "Stable dotnet/corefx packages no longer updated for .NET Core 3")

Reply:
Done